packages/flame_forge2d/example/lib/main.dart

```diff
@@ -68,6 +70,11 @@ void main() async {
       (DashbookContext ctx) => GameWidget(game: DraggableSample()),
       codeLink: link('draggable_sample.dart'),
     )
+    ..add(
+      'Basic joint',
+      (DashbookContext ctx) => GameWidget(game: JointSample()),
+      codeLink: link('joint_sample.dart'),
+    )
```

Comment:
Isn't this something different? Does it deserves its own entry on the CHANGELOG?